__tests__/cartItemsCount.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { dataController } from '../src/controller/data.controller';
```

Comment:
for future, put tests next to function/component being tested.

Reply:
remark accepted